src/lib/y2storage/partition.rb

```diff
@@ -244,6 +244,15 @@ def aligned_min_size(align_type = nil)
       min
     end
 
+    # Whether the partition belongs to an implicit partition table
+    #
+    # @see Y2Storage::PartitionTables::ImplicitPt
+    #
+    # @return [Boolean]
+    def implicit_partition_table?
```

Comment:
Why don't call this method simply `#implicit?`. Again, what is implicit is the partition, we are adding the partition table to the mix just because we need it (I don't think it's going to disappear in the short term, but still)... and `#implicit?` is shorter :-)